### ISSUE TITLE
VisDA2017 doesn't have category.txt

### ISSUE THREAD

**Issue by SivanDoveh** [kgl-prml/Contrastive-Adaptation-Network-for-Unsupervised-Domain-Adaptation#6]
Hi! 
can you please join the script you used in order to organize VisDA2017 in the way you mention?
the original VisDA2017 does not have a category.txt file
Thanks!

**Comment by kgl-prml** [comment]
Yes, it doesn't. I mean you may create the "category" file yourself and write the category names into it. Anyway, I will update an example in future.

**Comment by dongzhi0312** [comment]
UDA，target domain often no label，this paper support this case？
